.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,5 +9,22 @@ What issue is this PR targeting? If there is no issue that addresses the problem
  - [ ] review
  - [ ] adjust for comments
 
+## Code Review Checklist - author check these when done, reviewer verify
+ - [ ] Travis builds pass
+ - [ ] Code formatted with `scripts/format.sh`
+ - [ ] Changes have test coverage
+ - [ ] New exceptions, logging, errors - are messages distinct enough to track down in the code if they get thrown in production on non-debug builds?
+ - [ ] The PR is one logically integrated piece of work.  If there are unrelated changes, are they at least separate commits?
+ - [ ] Commit messages - are they clear enough to understand the intent of the change if we have to look at them later?
+ - [ ] Code comments - are there comments explaining the intent?
+ - [ ] Relevant docs updated
+ - [ ] Changelog entry if required (new features, bugfixes, behaviour changes)
```

Comment:
Where and what is our changelog? I admit I wouldn't know where/how to add a changelog entry.